src/ngraph/runtime/dynamic_wrapper/dynamic_wrapper_backend.hpp

```diff
@@ -0,0 +1,112 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <memory>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "ngraph/runtime/backend.hpp"
+#include "ngraph/runtime/host_tensor.hpp"
+#include "ngraph/runtime/tensor.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace dynamic_wrapper
+        {
+            class DynamicWrapperBackend;
+            class WrappedExecutable;
+            class WrappedStaticTensor;
```

Comment:
Not sure why we need a static tensor wrapper.

Reply:
I think you're right---my reason for adding this wound up being invalidated somewhere along the line. Let me take it out and see if anything breaks. :)